packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -132,6 +132,18 @@ function ViewGroupInner(
     setGroupImage,
   } = useGroup(accountId, chat)
 
+  useEffect(() => {
```

Comment:
This should not be neccessary, since the chat passed as prop to ViewGroup should be updated but it isn't!

MainScreen -> ChatHeading(chat) // chat.contacts updated after blocking/unblocking!!

useOpenViewGroupDialog -> openViewProfileDialog(chat)

useDialog -> openDialog(ViewGroup, chat)

DialogContext -> createElement(ViewGroup, ...additionalProps)

ViewGroup(chat)

ViewGroupInner(chat)  chat.contacts not updated!!